base/code/src/components/Navbar.tsx

```diff
@@ -0,0 +1,36 @@
+import { css } from 'emotion';
+import * as React from 'react';
+import { NavLink } from 'react-router-dom'
+import { LINK_COLOR } from '../consts/colors';
+
+const navbarClass = css`
+  height: 100px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 100px;
+`;
+
+const navigationLink = css`
```

Comment:
The class variables should be named consistently, so better to add the `Class` suffix here also